build.psm1

```diff
@@ -610,6 +610,22 @@ Fix steps:
         }
     }
 
+    if ($Environment.IsDebian9)
+    {
+        # add two symbolic links to system shared libraries that libmi.so is dependent on to handle
+        # platform specific changes. This appears to be a change in Debian 9; Debian 8 did not need these
+        # symlinks.
+        if (!(Test-Path -Path "$publishPath/libssl.so.1.0.0"))
```

Comment:
The code for this is very similar to the code for RedHatFamily. Can we combine logic like this:

```PowerShell

if($Environment.IsRedHatFamily -or $Environment.IsDebian9){
if($Environment.IsRedHatFamily) {
    $sslTarget = "/lib64/libssl.so.10"
    $cryptoTarget = "/lib64/libcrypto.so.10"
} elseif ($Enviroment.IsDebian9) {
 …..
}
   if(-not (Test-Path …) {
   }

  if(-not (Test-Path …) {
  }
}

Reply:
fixed